homeassistant/components/light/nanoleaf_aurora.py

```diff
@@ -17,13 +17,18 @@
 from homeassistant.util import color as color_util
 from homeassistant.util.color import \
     color_temperature_mired_to_kelvin as mired_to_kelvin
+from homeassistant.util.json import load_json, save_json
 
 REQUIREMENTS = ['nanoleaf==0.4.1']
 
 _LOGGER = logging.getLogger(__name__)
 
 DEFAULT_NAME = 'Aurora'
 
+DATA_NANOLEAF_AURORA = 'nanoleaf_aurora'
+
+CONFIG_FILE = 'nanoleaf_aurora.conf'
```

Comment:
If the config file is not supposed to be edited by users, we want it to be hidden, ie dart with a dot.